### ISSUE TITLE
Duplicate bind mount error

### ISSUE THREAD

**Issue by iMerica** [docker/compose#1628]
This appears when doing `docker-compose up -d` pretty frequently since upgrading to `1.3.1`.

**Comment by iMerica** [comment]
I think this is happening because Compose is creating two containers for the same service and there is a collision with the bind mounts.

**Comment by aanand** [comment]
Duplicate (hah) of https://github.com/docker/compose/issues/1573

**Comment by mahnunchik** [comment]
@aanand I think that it isn't a duplicate. I have the error:

```bash
$ docker-compose up -d
Removing blog_web_1...
blog_mysql_1 is up-to-date
Recreating b47774d8e1_blog_web_1...
Duplicate bind mount /app
```

/app are mounted by following config:
```yml
  volumes:
    - ./:/app/
```

**Comment by aanand** [comment]
OK, these are two different errors (name conflict vs "Duplicate bind mount").

@mahnunchik: What's the output of `docker inspect` on the image that your web container uses?

**Comment by mahnunchik** [comment]
```
$ docker inspect blog_web
[
{
    "Id": "af7094920fd2c001d887593c29e92eeb0daa04c574639d1364ec02719d4f588f",
    "Parent": "7d7de2f17ffdd2992490c3f7294f5ef1bace554646f26854cc28057847aa669b",
    "Comment": "",
    "Created": "2015-07-16T12:51:58.751721047Z",
    "Container": "22ea3eeafd717414379e32b93b1b54af8570715605f38cba7cf953604ae0f007",
    "ContainerConfig": {
        "Hostname": "b20c898782d0",
        "Domainname": "",
        "User": "",
        "AttachStdin": false,
        "AttachStdout": false,
        "AttachStderr": false,
        "PortSpecs": null,
        "ExposedPorts": {
            "80/tcp": {}
        },
        "Tty": false,
        "OpenStdin": false,
        "StdinOnce": false,
        "Env": [
            "PATH=/root/.composer/vendor/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
            "COMPOSER_HOME=/root/.composer"
        ],
        "Cmd": [
            "/bin/sh",
            "-c",
            "#(nop) EXPOSE 80/tcp"
        ],
        "Image": "7d7de2f17ffdd2992490c3f7294f5ef1bace554646f26854cc28057847aa669b",
        "Volumes": {
            "/app": {}
        },
        "VolumeDriver": "",
        "WorkingDir": "/app",
        "Entrypoint": null,
        "NetworkDisabled": false,
        "MacAddress": "",
        "OnBuild": [],
        "Labels": {}
    },
    "DockerVersion": "1.7.1",
    "Author": "Eugeny Vlasenko",
    "Config": {
        "Hostname": "b20c898782d0",
        "Domainname": "",
        "User": "",
        "AttachStdin": false,
        "AttachStdout": false,
        "AttachStderr": false,
        "PortSpecs": null,
        "ExposedPorts": {
            "80/tcp": {}
        },
        "Tty": false,
        "OpenStdin": false,
        "StdinOnce": false,
        "Env": [
            "PATH=/root/.composer/vendor/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
            "COMPOSER_HOME=/root/.composer"
        ],
        "Cmd": [
            "/root/run.sh"
        ],
        "Image": "7d7de2f17ffdd2992490c3f7294f5ef1bace554646f26854cc28057847aa669b",
        "Volumes": {
            "/app": {}
        },
        "VolumeDriver": "",
        "WorkingDir": "/app",
        "Entrypoint": null,
        "NetworkDisabled": false,
        "MacAddress": "",
        "OnBuild": [],
        "Labels": {}
    },
    "Architecture": "amd64",
    "Os": "linux",
    "Size": 0,
    "VirtualSize": 211989094
}
]

```

**Comment by mahnunchik** [comment]
I could not reproduce the sequence of actions which results in this situation but removing container named like `1b05f5a409_blog_web_1` solves the problem.

**Comment by aanand** [comment]
@mahnunchik I believe I've fixed it in #1787.
